packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -19,46 +21,23 @@ import trackerStepCss from "./TrackerStep.css";
 
 const withBaseName = makePrefixer("saltTrackerStep");
 
-type State = "default" | "completed";
-
-type StateWithActive = State | "active";
+type StageOptions = "pending" | "completed";
+type StatusOptions = "warning" | "error" | undefined;
 
 export interface TrackerStepProps extends ComponentPropsWithoutRef<"li"> {
   /**
-   * The state of the TrackerStep
+   * The slug of the icon to be displayed in the TrackerStep
```

Comment:
Not too sure i understand what this means in the props table

Reply:
Good spot. Have updated 👍